lib/libmpa/mpa_io.c

```diff
@@ -100,7 +101,8 @@ static int __mpa_digit_value(int c)
  *  Returns the maximum number of words needed to binary represent a number
  *  consisting of "digits" digits and each digits is in base "base".
  */
-static mpa_word_t __mpa_digitstr_to_binary_wsize_base_16(int digits)
+static
```

Comment:
Yeah, personally I'd prefer that. I wonder what checkpatch would prefer though. Or perhaps just shorten the function name, I bet no one but Patrik (the original author) is happy with that. :-)